Resources/Prototypes/_NF/Entities/Mobs/NPCs/mob_hostile_expeditions_aberrant_flesh.yml

```diff
@@ -10,24 +11,12 @@
   - type: NpcFactionMember
     factions:
     - AberrantFleshExpeditionNF
-  - type: HTN
-    rootTask:
-       task: SimpleHostileCompound
-    blackboard:
-      NavClimb: !type:Bool
-        true
-      NavInteract: !type:Bool
-        true
-      NavPry: !type:Bool
-        true
-      NavSmash: !type:Bool
-        true
```

Comment:
For whatever reason, when this is removed, the Clamp enemy becomes an unhittable, stationary monster that can't be highlighted.  Not quite sure why that is, need to look into this more.

Reply:
I think this can only happen when you spawn Clamps through console command while not in aghost. 

https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/85596e99-e325-4363-ae92-2b286c2befff

Reply:
Doesn't seem to happen when you use spawner for aberrant flesh and doesn't seem to happen on expeditions (jumped on 2 expeds with aberrant flesh)

Reply:
https://github.com/new-frontiers-14/frontier-station-14/assets/65374927/c0608e04-ff7f-493d-8b1b-867b64961824

Reply:
Uploading 2024-06-19 08-06-27.mp4…